src/core/src/enclave_sgx.rs

```diff
@@ -79,7 +79,8 @@ impl EnclaveConfig {
         let spid_str = str::from_utf8(&spid_vec).unwrap();
 
         let mut lib_path = env::current_dir().unwrap();
-        lib_path.push("../build/bin/libpoet_enclave.signed.so");
+        lib_path.push("/project/sawtooth-poet2/src/build/bin/libpoet_enclave.signed.so");
```

Comment:
Need to be fixed up in implementation rather than here. Need not rework now.

Reply:
This is just a workaround. Otherwise Jenkins job will fail and so the debian package creation.